src/sentry/utils/redis.py

```diff
@@ -100,7 +100,9 @@ def execute_command(self, *args, **kwargs):
 
 class _RedisCluster(object):
     def supports(self, config):
-        return config.get("is_redis_cluster", False)
+        # Default RedisCluster works on both a singular Redis instance
```

Comment:
What does "Default RedisCluster" mean in this comment? Is this referencing this class (`_RedisCluster`), or `StrictRedisCluster` or something else entirely? I'm not sure how to interpret this.

Is this trying to say that this will use a (single) Redis server by default (since `is_redis_cluster` defaults to `False`), but can also be expanded to use Redis Cluster (of any size) by setting the `is_redis_cluster` configuration key?

Reply:
Oh, yes, it's referencing `_RedisCluster`. I'll reword this.

And yes, your assessment is correct. This class now works with a singular redis server not in cluster mode, as well as a redis-cluster configured with `is_redis_cluster`. So we have one unified API to work with for "redis + redis-cluster" and that abstraction is something you don't have to really be concerned with. It'll make more sense to rename all of this stuff once `rb` is gone since we're really overloading the term "cluster" in all of this file.

Reply:
The updated comment is better, thanks.